### PR TITLE
feat: #306 Add tags to jobs and plans

### PR DESCRIPTION
**Pull request by Ekaterina1994**

### CODE REVIEW

File: mw-webapp/src/logic/wayPage/reportsTable/ModalContentJobTags.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+ */
+export const ModalContentJobTags = (props: JobDoneTagsProps) => {
+  const [jobTagsUpdated, setJobTagsUpdated] = useState<string[]>(props.jobDoneTags);
+
+  const allTags = Array.from(new Set(props.jobTags.concat(props.jobDoneTags).filter((tag) => tag !== DEFAULT_TAG)));
+
+  /**
+   * Remove job tag from Job done
+   */
+  const removeJobTagFromJobDone = async (jobTagToRemove: string) => {
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/reportsTable/ModalContentJobTags.tsx:49]: async? everywhere

File: mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx [hunk 2]
```diff
@@ -118,6 +100,34 @@ export const Columns = (props: ColumnsProps) => {
   const isOwner = user?.uuid === ownerUuid;
   const isMentor = !!user && !!user.uuid && props.way.mentors.has(user.uuid);
   const isUserOwnerOrMentor = isOwner || isMentor;
+  const [dayReports, setDayReports] = useState<DayReport[]>(props.way.dayReports);
+
+  /**
+   * Update DayReport
+   */
+  const updateReport = async (report: Partial<DayReport> & {
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx:108]: generic partialWithUuid

File: mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx [hunk 3]
```diff
@@ -21,12 +21,12 @@ export const TagStats = (props: TagStatsProps) => {
     <Fragment key={tagStat.name}>
       <StatisticLine
         description={`${tagStat.name} jobs finished:`}
-        value={`${tagStat.totalAmount} / ${tagStat.totalAmountPercentage} %`}
+        value={`${tagStat.totalAmount} (${tagStat.totalAmountPercentage} %)`}
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/wayStatistics/TagStats.tsx:24]:
```suggestion
        value={`${tagStat.totalAmount} (${tagStat.totalAmountPercentage}%)`}
```

File: mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx (hunk 2 above)
Comment (Ekaterina1994) [reply mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx:108]: added comment TODO

**Review by SergioAJS — APPROVED** [review]

**Pull request merged**